v0.3/tutorials/quickstart 1 button stopwatch.eve

```diff
@@ -0,0 +1,198 @@
+---
+layout: default
+title: Quickstart + One Button Stopwatch
+---
+
+{% raw %}
+
+# Quickstart + One Button Stopwatch
+
+This guide expands on the quickstart `Clock` page and shows you how to turn the clock into a one button stopwatch. The following code is meant to replace all the code from the original clock.
+
+## Implement system time and a race timer
+
+```
+commit
+  [#time #system/timer resolution: 10]
+  [#race-timer #stopped #reset]
+```
+
+Stopwatches tend to measure seconds to two decimal points, so our resolution changes to 10ms (or one centisecond) to account for that. The other thing added here is a `#race-timer` record that lets us keep track of when the stopwatch is stopped or running or has been reset back to zero.
+
+
+## Draw clock face
+
+```
+search
+  [#app page: "Clock"]
+  view = [#qs-contents]
+  [#race-timer minute second millisecond]
+
+bind
+  view.children += [#svg/root sort: 1 viewBox: "0 0 100 100", width: "300px", children:
+    [#svg/circle #second-face sort: 1, cx: 50, cy: 50, r: 48, fill: "#fff", stroke: "#000", stroke-width: 0.5]
+    [#svg/circle #minute-face sort: 2, cx: 50, cy: 26, r: 12, fill: "#fff", stroke: "#000", stroke-width: 0.25]
+    [#minute-hand sort: 3, degrees: 6 * minute, length: 11, stroke: "#000", stroke-width: 0.5]
+    [#second-hand sort: 4, degrees: 6 * second + 0.006 * millisecond, length: 46, stroke: "#ff0000", stroke-width: 0.5]]
+```
+
+This block closely resembles its predecessor, but the search looks for `#race-timer` now instead of the system time since it's the race time that will determine the position of the hands.
```

Comment:
it's unclear to me what that description adds here. the system timer is ticking at the same speed as the race timer; the race timer is just counting from 0 as opposed to jan 1 1970